tests/kafkatest/services/security/security_config.py

```diff
@@ -73,7 +97,9 @@ class SecurityConfig(TemplateRenderer):
     KRB5CONF_PATH = "/mnt/security/krb5.conf"
     KEYTAB_PATH = "/mnt/security/keytab"
 
-    ssl_stores = Keytool.generate_keystore_truststore('.')
+    ssl_stores = SslStores()
```

Comment:
This probably shouldn't be a static class variable if it's being overridden in some tests via the class (i.e. using `SecurityConfig.ssl_stores = ...` rather than overriding on a per-instance basis, i.e. `self.ssl_stores = ...` in the case where it needs to be overridden).